autogpt/commands/execute_code.py

```diff
@@ -97,6 +97,30 @@ def execute_python_file(filename: str) -> str:
         return f"Error: {str(e)}"
 
 
+def validate_command(command: str) -> bool:
+    """Validate a command to ensure it is allowed
+
+    Args:
+        command (str): The command to validate
+
+    Returns:
+        bool: True if the command is allowed, False otherwise
+    """
+    tokens = command.split()
+
+    if not tokens:
+        return False
+
+    if CFG.deny_commands and tokens[0] not in CFG.deny_commands:
+        return False
+
+    for keyword in CFG.allow_commands:
+        if keyword in tokens:
+            return True
```

Comment:
Now looks better, but it is still missing logic - if you have allowlist (not empty) and it doesnt find the command in the allowlist it must return False.

Reply:
I feel like there should be a prettier way than I did...